### PR TITLE
When anon flag on always show my ft nav item

### PR DESCRIPTION
**Pull request by adamsilver**

### CODE REVIEW

File: src/js/account.js [hunk 1]
```diff
@@ -15,6 +16,10 @@ function init() {
 				throw err;
 			}, 0);
 		});
+
+	if(flags.get('anonymousMyFt')) {
+		document.documentElement.setAttribute('data-next-anonymous-myft', 'on');
```
Comment (keirog) [inline src/js/account.js]: You could grab this from data-next-flags in the <html> instead

Comment (adamsilver) [inline src/js/account.js]: Yep, doing. Thank you sir.

**Comment by adamsilver** [comment]
Had to put original code back, the flag status' only show when off, and this navigation item shouldn't rely on the feature flag in order to show it.

**Comment by keirog** [comment]
This won't work for [core-experience users](http://financial-times.github.io/next/docs/developer-guide/browser-support/) :-(

**Comment by adamsilver** [comment]
Yep, sign in/out doesn't work in core experience yet.  When that is done,
we will refactor this.

On 2 June 2015 at 15:21, Keith Rogers <notifications@github.com> wrote:

> This won't work for core-experience users
> <http://financial-times.github.io/next/docs/developer-guide/browser-support/>
> :-(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Financial-Times/n-header-footer/pull/4#issuecomment-107971220>
> .
>

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.

**Pull request merged**